geode-cq/src/test/java/org/apache/geode/cache/query/cq/internal/ServerCQResultsCacheReplicateRegionImplTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.cache.query.cq.internal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Future;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.apache.geode.test.junit.rules.ExecutorServiceRule;
+
+public class ServerCQResultsCacheReplicateRegionImplTest {
+  @Rule
+  public ExecutorServiceRule executorService = new ExecutorServiceRule();
+
+  private final ServerCQResultsCacheReplicateRegionImpl serverCQResultCache =
+      new ServerCQResultsCacheReplicateRegionImpl();
+  private final ArrayList<String> objects = new ArrayList<>();
+  private final Random random = new Random(1188999);
```

Comment:
Is there a reason that this random is using a fixed seed?

Reply:
Originally I was trying to see if the test written can fail repeatably, so same random seed can make the test more likely to fail repeatedly. Once fix is there, I think random seed may be removed. But it should not matter in my opinion.